promql/promqltest/testdata/aggregators.test

```diff
@@ -235,21 +235,25 @@ eval instant at 0m max(http_requests)
   {} 4
 
 # The histogram is ignored here so the result doesn't change but it has an info annotation now.
-eval_info instant at 0m max({job="api-server"})
+eval instant at 0m max({job="api-server"})
+expect info
```

Comment:
Could you please add a tab here, and in all other places, to align this line with the subsequent data lines?

Reply:
Fixed the indentation in [d269b0f](https://github.com/prometheus/prometheus/commit/d269b0fe705e229f0fa9dc6dc0e00b322b0c21e4).